stratum/hal/lib/common/yang_parse_tree_paths.cc

```diff
@@ -3587,15 +3621,18 @@ void YangParseTreePaths::AddSubtreeNode(const Node& node, YangParseTree* tree) {
   TreeNode* tree_node = tree->AddNode(GetPath("debug")("nodes")(
       "node", node.name())("packet-io")("debug-string")());
   SetUpDebugNodesNodePacketIoDebugString(node.id(), tree_node, tree);
-  tree_node = tree->AddNode(GetPath("components")("component", node.name())
-      ("integrated-circuit")("config")("node-id")());
-  SetUpComponentsComponentIntegratedCircuitConfigNodeId(node.id(),
-                                                        tree_node, tree);
-
-  tree_node = tree->AddNode(GetPath("components")("component", node.name())
-      ("integrated-circuit")("state")("node-id")());
-  SetUpComponentsComponentIntegratedCircuitStateNodeId(node.id(),
-                                                       tree_node, tree);
+  tree_node = tree->AddNode(GetPath("components")(
+      "component", node.name())("integrated-circuit")("config")("node-id")());
+  SetUpComponentsComponentIntegratedCircuitConfigNodeId(node.id(), tree_node,
+                                                        tree);
+  tree_node = tree->AddNode(GetPath("components")(
+      "component", node.name())("integrated-circuit")("state")("node-id")());
+  SetUpComponentsComponentIntegratedCircuitStateNodeId(node.id(), tree_node,
+                                                       tree);
+  tree_node = tree->AddNode(GetPath("components")(
+      "component", node.name())("integrated-circuit")("state")("part-no")());
+  SetUpComponentsComponentIntegratedCircuitStatePartNo(node.id(), tree_node,
+                                                       tree);
```

Comment:
We can add `augment` to the Stratum model(`openconfig-platform-stratum.yang`) since paths described here do not exist.
https://github.com/openconfig/public/blob/master/release/models/platform/openconfig-platform.yang#L688-L702
http://www.yang-central.org/twiki/pub/Main/YangDocuments/draft-ietf-netmod-yang-02.html#rfc.section.4.2.8